go/chat/sender.go

```diff
@@ -635,7 +621,7 @@ func (s *BlockingSender) Send(ctx context.Context, convID chat1.ConversationID,
 			ChannelMention: chanMention,
 			TopicNameState: topicNameState,
 		}
-		plres, err = ri.PostRemote(ctx, rarg)
+		plres, err = s.getRi().PostRemote(ctx, rarg)
```

Comment:
same as above, what if `nil`?

Reply:
Also this is the change that matters for your situation, the ones above I just cleaned up (but they had no bugs)